### DISCUSSION TITLE
How can i go to the next or previous point on a track?

### DISCUSSION THREAD

**Discussion by wega789** [osmandapp/OsmAnd#19886] [Q&A]
My problem is, that I - like others - often forget to stop/pause recording on arrival or intermediate stops. This can produce a cloud of dozens of points and it is almost impossible to select the first point along the track within this cloud by tipping on the map. This is needed for trimming or creating a new segment. The reason is that not only the points are very close but also the connection lines are scattered all over the place.

Is there a workflow to handle this situation? I could think of selecting one point outside the cloud and walk from there point by point. But there is no action or menu to go to the next/previous point. Of course I could check the point number of such an outside point and find the number of the next one. But then I couldn't find a way to position on a point by number because the point list is not searchable but only scrollable, which takes quite a while for point 1075.

Thank you for any suggestions

**Comment by pebogufi** [comment]
Avoid creating dozends/hundrets of useless points during a stop or pause (drinking a coffee. ..) is easy with this setting:
select your profile / trip recording 
In section Logging accuracy set
Minimum displacement 10m
Minimum accuracy 50m
Minimum speed 2 km/h
Of cource other values may also do it.
Theese are just MY values. When making a hiking tour or bike tour and stopping for a break ( may be 30 min. or more) I NEVER stop or pause recording and have very few unwanted points with this setting.
Just try it, play around with the values if you want.
